FNML/definitions.md

```diff
@@ -3,7 +3,9 @@
 - <dfn>FnO</dfn>: the Function Ontology. Describes implementation-independent functions' inputs and outputs, and execution descriptions.
 - <dfn>RML</dfn>: RDF Mapping Language. Describes schema transformations from heterogeneous data sources to RDF.
 - An <dfn>RML mapping</dfn> is a set of mapping rules, described in <a>RML</a>.
-- An <dfn>RML processor</dfn> is a tool that interprets the <a>RML mapping</a> and executes its rules to generate a knowledge graph.
+- A <dfn>triples map</dfn> specifies the rules for translating input data to zero or more RDF triples, as [defined](https://rml.io/specs/rml/#triples-map) in the RML specification.
```

Comment:
> The triples map is a rule that maps each iteration in the logical source to a number of RDF triples.

according to https://github.com/kg-construct/rml-fno-spec/blob/main/R2RML-core/overview.md

Reply:
(subject to discussion for refinement)